pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -739,7 +739,7 @@ def cmd_instance_enumerate(context, classname, options):
         display_cim_objects(context, results, context.output_format,
                             summary=options['summary'])
 
-    except Error as er:
+    except (Error, ValueError) as er:
```

Comment:
A ValueError indicates a programming error in the vast majority of cases. A programming error should result in a traceback display and not in a one-line error message that presumably will contain no text other than "ValueError".

If you want to transform a specific occurrence of ValueError into an error message, thenplease do it directly on the statement where the ValueError occurs, raise a pywbemtools.Error of some sort, and give it a reasonable error message text.

Reply:
But this is not my error, it is what we are returning from the pywbem Iter... methods if we do an iter... with FilterQuery and force/ default to the traditional operations.  We would have to change pywbem to correct that.

Reply:
ok